pkg/resources/reconciler.go

```diff
@@ -142,9 +142,9 @@ func (r *Reconciler) ReconcilePullSecret(ctx context.Context, namespace, secretN
 	return v1alpha1.PhaseCompleted, nil
 }
 
-func (r *Reconciler) ReconcileSubscription(ctx context.Context, owner ownerutil.Owner, t marketplace.Target, client pkgclient.Client, maxVersion *Version) (v1alpha1.StatusPhase, error) {
+func (r *Reconciler) ReconcileSubscription(ctx context.Context, owner ownerutil.Owner, t marketplace.Target, targetNS string, client pkgclient.Client, maxVersion *Version) (v1alpha1.StatusPhase, error) {
```

Comment:
is this change to allow an operator to watch a namespace different to the one it's installed in?

Reply:
Yes exactly

Reply:
Cool